ui-mockups/src/scss/style.scss

```diff
@@ -5,84 +5,126 @@ $ah-alternate-color: #FFDE00;
 $ah-secondary-text-color: #999999;
 
 
-body{
+body {
     background: $bg-color;
 }
 
-.btn-link{
+.btn-link {
     color: $ah-primary-color;
 }
 
-.btn-link:hover{
+.btn-link:hover {
     color: $ah-primary-color;
     text-decoration: none;
 }
 
 // Card
-.card-body{
+.card-body {
     background-color: $bg-color;
 }
 
-.card{
+.card {
     border: none;
 }
 
-.card-text{
+.card-text {
     color: $ah-secondary-text-color;
     font-size: 10pt;
 }
 
-small a, .ah-author-link{
+small a,
+.ah-author-link {
     color: $ah-primary-color;
     text-decoration: underline;
 }
-small a:hover, .ah-author-link:hover{
+
+small a:hover,
+.ah-author-link:hover {
     text-decoration: none;
     color: $ah-primary-color;
-    background-color: $ah-alternate-color;
```

Comment:
Please maintain the background color. I find changing the font size on hover too dramatic for the eye

Reply:
Done